src/main/java/org/alliancegenome/curation_api/model/entities/DiseaseAnnotation.java

```diff
@@ -127,7 +127,7 @@
     @IndexingDependency(reindexOnUpdate = ReindexOnUpdate.SHALLOW)
     @ManyToOne
     @JsonView({View.FieldsOnly.class})
-    private BiologicalEntity diseaseGeneticModifier;
+    private GenomicEntity diseaseGeneticModifier;
```

Comment:
This will need a migration?

Reply:
@oblodgett - I'm not sure it needs one.  The column name in the diseaseannotation table remains diseasegeneticmodifier_curie and presumably Hibernate will take care of pointing the foreign key to genomicentity rather than biologicalentity?